src/pytorch_lightning/accelerators/cpu.py

```diff
@@ -70,22 +73,9 @@ def register_accelerators(cls, accelerator_registry: Dict) -> None:
         )
 
 
-# CPU device metrics
-_CPU_VM_PERCENT = "cpu_vm_percent"
-_CPU_PERCENT = "cpu_percent"
-_CPU_SWAP_PERCENT = "cpu_swap_percent"
-
-
 def get_cpu_stats() -> Dict[str, float]:
-    if not _PSUTIL_AVAILABLE:
-        raise ModuleNotFoundError(
-            "Fetching CPU device stats requires `psutil` to be installed."
-            " Install it by running `pip install -U psutil`."
-        )
-    import psutil
-
-    return {
-        _CPU_VM_PERCENT: psutil.virtual_memory().percent,
-        _CPU_PERCENT: psutil.cpu_percent(),
-        _CPU_SWAP_PERCENT: psutil.swap_memory().percent,
-    }
+    rank_zero_deprecation(
+        "`pytorch_lightning.accelerators.cpu.get_cpu_stats` has been deprecated in v1.8.0 and will be removed in"
+        " v1.10.0. Please use `lightning_lite.accelerators.cpu.get_cpu_stats` instead."
+    )
+    return new_get_cpu_stats()
```

Comment:
I think all the stats logic should stay in PL, removed from the Lite implementations.

This is for the DeviceStats callback, and Lite won't have callbacks